docker-registry-sync/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN pip3 install --upgrade \
 
 RUN mkdir -p /app
 
-COPY reposync /app/reposync
+COPY . /app/reposync
```

Comment:
TIP: this will copy everything inside inclusive .md files (since the root .dockerignore file does not start with *).
Actually the good practices should be that the root .dockerignore has * as first line, thus not copying anything inside the docker container -> that makes docker building much faster. then you can ! ignore files that you need inside a .dockerignore file in docker-registry-sync folder.